src/System.CommandLine/EnvironmentVariablesDirective.cs

```diff
@@ -26,27 +26,24 @@ private sealed class EnvironmentVariablesDirectiveAction : CliAction
         {
             private readonly EnvironmentVariablesDirective _directive;
 
-            internal EnvironmentVariablesDirectiveAction(EnvironmentVariablesDirective directive) => _directive = directive;
+            internal EnvironmentVariablesDirectiveAction(EnvironmentVariablesDirective directive)
+            {
+                _directive = directive;
+                Exclusive = false;
+            }
 
             public override int Invoke(ParseResult parseResult)
             {
                 SetEnvVars(parseResult);
 
-                return parseResult.CommandResult.Command.Action?.Invoke(parseResult) ?? 0;
+                return 0;
             }
 
             public override Task<int> InvokeAsync(ParseResult parseResult, CancellationToken cancellationToken = default)
             {
-                if (cancellationToken.IsCancellationRequested)
-                {
-                    return Task.FromCanceled<int>(cancellationToken);
-                }
-
```

Comment:
I know that it's very unlikely for the cancellation to be requested so quickly, but it's possible, so we should keep the checks and ensure that all actions handle cancellation properly.

In theory we could move this logic to the `InvocationPipeline`, but it would not cover a scenario where the users do `parseResult.Action.InvokeAsync(token)` rather than the usual `parseResult.InvokeAsync(token)`

Reply:
Related, I see we have a few clones of this code: 

https://github.com/dotnet/command-line-api/blob/72ace160fa6471081c7092ff548d5fd0f14578c3/src/System.CommandLine/Help/HelpOptionAction.cs#L33-L36

I'd like to find a way to prevent `CliAction` implementations from incidentally differing from one another, because we are testing them as if they're all the same.